db/schema.rb

```diff
@@ -55,8 +55,8 @@
     t.text     "body"
     t.integer  "user_id"
     t.integer  "category_id"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.datetime "created_at"
```

Comment:
I see you've taken out the null: false portion of the schema here. Is there a migration that didn't get run out of a previous PR that lost this somewhere or was it removed intentionally?